pipelines/controller/src/main/java/com/google/fhir/analytics/DwhDto.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020-2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.fhir.analytics;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
```

Comment:
In most places in our code, we use Gson for JSON related functionality; I suppose in this case we have to rely on Jackson because that is what is used by Spring (for converting objects to JSON), correct?

Reply:
Yes Jackson is the default but if we needed to use Gson we could do it with some extra configuration.